### ISSUE TITLE
Failure when using regexs in files array

### ISSUE THREAD

**Issue by ghost** [mape/connect-assetmanager#6]
Getting this error when I use RegExp in the files array, even as indicated in the document examples:

```
TypeError: Object /jquery.*/ has no method 'match'
    at /usr/lib/node/.npm/connect-assetmanager/0.0.18/package/lib/assetmanager.js:92:15
    at Array.forEach (native)
    at /usr/lib/node/.npm/connect-assetmanager/0.0.18/package/lib/assetmanager.js:91:17
    at /usr/lib/node/.npm/connect-assetmanager/0.0.18/package/lib/assetmanager.js:18:6
    at Array.forEach (native)
    at Object.forEach (/usr/lib/node/.npm/connect-assetmanager/0.0.18/package/lib/assetmanager.js:16:22)
    at Function.<anonymous> (/usr/lib/node/.npm/connect-assetmanager/0.0.18/package/lib/assetmanager.js:89:12)
    at next (/usr/lib/node/.npm/step/0.0.4/package/lib/step.js:51:23)
    at check (/usr/lib/node/.npm/step/0.0.4/package/lib/step.js:73:14)
    at /usr/lib/node/.npm/step/0.0.4/package/lib/step.js:86:20
```

Which makes sense - the code at that point is assuming all entries are strings (marker is line 92 of assetmanager.js):

```
function(err, contents) {
                settings.forEach(function (group, groupName) {
                        if (!group.stale) {
                                group.files.forEach(function (file) {
------>                              if (file.match(/^http:\/\//)) {
                                                return;
                                        }
                                        fs.watchFile(group.path + file, function (old, newFile) {
                                                if (old.mtime.toString() != newFile.mtime.toString()) {
                                                        self.generateCache(groupName);
                                                }
                                        });
                                });
                        }
                });
                self.generateCache();
        });
```

Finally - this bit in the Readme doesn't immediately make sense, can you clarify?

```
If you want to add all files from the path supplied add ''. It will insert the files at the position of the . 
```

**Comment by tixelated** [comment]
(The last bit about the Readme was just a markdown error, that part makes perfect sense now.)

**Comment by edwardmsmith** [comment]
I'm having this issue as well when connect-assetmanager is installed as an npm dependency in our project.

**Comment by edwardmsmith** [comment]
Ah, my bad.  The issue I was having actually sprung from the new npm.  When I ran 'npm install' on my package, less (the stylesheet compiler) was being installed locally - therefore 'lessc' was not in the path, and my Cakefile was not successfully compiling the stylesheets.  The result was that I got the a similar error to the OP, due to the file not being on the filesystem.

Tixelated - if you have not solved this, be sure that there's a "jquery.{anything}" file where assetmanager would be looking for it.
